src/structures/paging/page_table.rs

```diff
@@ -176,7 +184,7 @@ pub struct PageTable {
 
 impl PageTable {
     /// Creates an empty page table.
-    pub const fn new() -> Self {
+    pub fn new() -> Self {
```

Comment:
This unconditionally makes `PageTable::new` non-const.

Reply:
oops

Reply:
fixed